### PR TITLE
[MIG] medical_base_us: Migrate to 10.0

### PR DESCRIPTION
**Pull request by kellylougheed**
Depends:
- [x] #119

### CODE REVIEW

File: medical_base_us/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    'summary': 'Provides some helper mixins for US locale',
+    'version': '10.0.1.0.0',
+    'author': 'LasLabs, Odoo Community Association (OCA)',
+    'category': 'Medical',
+    'website': 'https://laslabs.com',
+    'license': 'AGPL-3',
+    'installable': True,
+    'auto_install': False,
+    'depends': [
+        'medical',
```
Comment (lasley) [inline medical_base_us/__manifest__.py:16]: This is how you can tell what your PR depends on

**Comment by lasley** [comment]
Rebased now that depend has been merged

**Review by lasley — APPROVED** [review]
LGTM thanks @kellylougheed

**Review by lasley — CHANGES_REQUESTED** [review]
Oops just looked at the commit message, looks like some other stuff got squashed in the description. Please amend the commit to fix the message

We're also missing some history. Best way to get it back is to
* check this branch out, 
* reset --hard to OCA/10.0, 
* run the format-patch from OCA/10.0..OCA/9.0 (to get our lost history),
* then cherry pick the b03ab16d6efd63857b9012679f2dd33f9ca7bb40 commit from here choosing to prefer their's (the cherry picked) over your's (9.0 filtered), 
* then fix the message

**Comment by lasley** [comment]
@kellylougheed - is this ready for review?

**Comment by kellylougheed** [comment]
@lasley yes!

**Comment by lasley** [comment]
@kellylougheed - https://github.com/OCA/vertical-medical/pull/156#pullrequestreview-16701357 is still unattended

**Comment by kellylougheed** [comment]
Whoops, I thought I had done that! Just went through all the steps in your review @lasley

File: medical_base_us/README.rst [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+`here <https://github.com/OCA/vertical-medical/issues/new?body=module:%20medical_base_us%0Aversion:%2010.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
+
+
+Credits
+=======
+
+Contributors
+------------
+
+* Dave Lasley <dave@laslabs.com>
```
Comment (lasley) [inline medical_base_us/README.rst:45]: Add yourself

File: medical_base_us/README.rst [hunk 3]
```diff
@@ -0,0 +1,59 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
```
Comment (lasley) [inline medical_base_us/README.rst:1]: Please update this to the new ReadMe template https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst

File: medical_base_us/__openerp__.py [hunk 4]
```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
```
Comment (lasley) [inline medical_base_us/__openerp__.py]: Remove this file

File: medical_base_us/models/medical_abstract_dea.py [hunk 5]
```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 LasLabs Inc.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import models, api
```
Comment (lasley) [inline medical_base_us/models/medical_abstract_dea.py]: Alphabetize imports

File: medical_base_us/models/medical_abstract_dea.py [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+import re
```
Comment (lasley) [inline medical_base_us/models/medical_abstract_dea.py]: stdlib packages should be imported above odoo packages. Import ordering - https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#imports

File: medical_base_us/models/medical_abstract_dea.py [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+        3. 16 * 2 = 32
+        4. 15 + 32 = 47
+
+        Params:
+            dea_num (str): DEA ID to validate
+        Returns:
+            bool
+        """
+
+        if len(dea_num) != 9:
```
Comment (lasley) [inline medical_base_us/models/medical_abstract_dea.py]: `len(str(dea_num))`

**Review by lasley — CHANGES_REQUESTED** [review]

**Comment by kellylougheed** [comment]
Changes made @lasley

**Comment by obulkin** [comment]
This PR has been superseded by https://github.com/OCA/vertical-medical/pull/196 and can be closed.

**Pull request closed without merging**